mlx/ops.cpp

```diff
@@ -1785,15 +1785,6 @@ array sort(const array& a, int axis, StreamOrDevice s /* = {} */) {
     throw std::invalid_argument(msg.str());
   }
 
-  // TODO: Fix GPU kernel
```

Comment:
Is worth a warning here for sorts greater than maxint?

Reply:
`array::shape` is made of ints, so each of those will be `<= INT_MAX`

Reply:
Ah good point -- nice!